src/Http/Controllers/SeatGroupsController.php

```diff
@@ -22,8 +22,9 @@ class SeatGroupsController extends Controller
      */
     public function index()
     {
-            return view('seatgroups::index')
-                ->with('seatgroups',SeatGroup::all());
+        $seatgroups = SeatGroup::all();
+
+        return view('seatgroups::index', compact('seatgroups'));
```

Comment:
i see this pattern often in SeAT. What is the reason behind the compact? Does it save data volume? computing power?